### PR TITLE
Update tool box icon spec

### PR DESCRIPTION
**Pull request by kaiguo**

### CODE REVIEW

**Comment by karann-msft** [comment]
### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/guides/Create-UI-Controls.md](https://github.com/kaiguo/docs.microsoft.com-nuget/blob/patch-1/docs/guides/Create-UI-Controls.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/nuget/guides/create-ui-controls?branch=pr-en-us-1149) | [Details](#user-content-docs%2Fguides%2FCreate-UI-Controls.md)

<a name="docs%2Fguides%2FCreate-UI-Controls.md"></a>
### [docs/guides/Create-UI-Controls.md](https://github.com/kaiguo/docs.microsoft.com-nuget/blob/patch-1/docs/guides/Create-UI-Controls.md)
 - **Line 89**: **[Warning]** Invalid file link:(~/guides/media/).

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C11%5C8%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C201811080107218369-1149%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=kctxLSfh1Lhbyl4SX%2Fc2gvMZmE9EqNEN8wIcx89zMbI%3D&st=2018-11-08T01%3A03%3A21Z&se=2018-12-09T01%3A08%3A21Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/guides/Create-UI-Controls.md [hunk 1]
```diff
@@ -84,7 +84,11 @@ In the following example, the control implemented in `ManagedPackage.winmd` will
 
 To display a custom icon in the toolbox/assets pane, add an image to your project or the corresponding `design.dll` project with the name “Namespace.ControlName.extension” and set the build action to “Embedded Resource”. You must also ensure that the associated `AssemblyInfo.cs` specifies the ProvideMetadata attribute - `[assembly: ProvideMetadata(typeof(RegisterMetadata))]`. See this [sample](https://github.com/NuGet/Samples/blob/master/ExtensionSDKasNuGetPackage/NativePackage.Design/Properties/AssemblyInfo.cs#L20).
 
-Supported formats are `.png`, `.jpg`, `.jpeg`, `.gif`, and `.bmp`. The recommended image size is 64 pixels by 64 pixels.
+Supported formats are `.png`, `.jpg`, `.jpeg`, `.gif`, and `.bmp`. The recommended format is BMP24 in 16 pixels by 16 pixels.
+
+![Tool box icon sample](media/)
```
Comment (kaiguo) [inline docs/guides/Create-UI-Controls.md]: @karann-msft I didn't find this `media` directory, can you put one of the icons in our email thread here as example? Thanks!

**Comment by karann-msft** [comment]
### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/guides/Create-UI-Controls.md](https://github.com/kaiguo/docs.microsoft.com-nuget/blob/patch-1/docs/guides/Create-UI-Controls.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/nuget/guides/create-ui-controls?branch=pr-en-us-1149) | [Details](#user-content-docs%2Fguides%2FCreate-UI-Controls.md)

<a name="docs%2Fguides%2FCreate-UI-Controls.md"></a>
### [docs/guides/Create-UI-Controls.md](https://github.com/kaiguo/docs.microsoft.com-nuget/blob/patch-1/docs/guides/Create-UI-Controls.md)
 - **Line 89**: **[Warning]** Invalid file link:(~/guides/media/).

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C11%5C8%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C201811082236558378-1149%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=Rlj00nneH3uWZbkXnJm8gQLMNznCKRzFBsp6iVT4Nm8%3D&st=2018-11-08T22%3A32%3A53Z&se=2018-12-09T22%3A37%3A53Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/guides/Create-UI-Controls.md (hunk 1 above)
Comment (kaiguo) [reply docs/guides/Create-UI-Controls.md]: I did find the folder but I wasn't able to push? Permission issue?

**Comment by karann-msft** [comment]
@kaiguo thanks for the PR! I have added an image and will merge it shortly.

**Comment by karann-msft** [comment]
### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/guides/Create-UI-Controls.md](https://github.com/kaiguo/docs.microsoft.com-nuget/blob/patch-1/docs/guides/Create-UI-Controls.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/nuget/guides/create-ui-controls?branch=pr-en-us-1149) | [Details](#user-content-docs%2Fguides%2FCreate-UI-Controls.md)

<a name="docs%2Fguides%2FCreate-UI-Controls.md"></a>
### [docs/guides/Create-UI-Controls.md](https://github.com/kaiguo/docs.microsoft.com-nuget/blob/patch-1/docs/guides/Create-UI-Controls.md)
 - **Line 89**: **[Warning]** Invalid file link:(~/guides/media/ColorPicker_16x16x24.bmp).

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C11%5C9%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C201811091951501576-1149%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=%2FCWOL25JwvcbJlq1XiGIRM1FCCbYVD9eBZ4OGG%2B%2FUJs%3D&st=2018-11-09T19%3A47%3A52Z&se=2018-12-10T19%3A52%3A52Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**